### PR TITLE
Adds super cool powerloss sound to grid check event

### PR DESCRIPTION
**Pull request by Ty-Omaha**
**What does this PR do:**
Plays: https://raw.githubusercontent.com/Ty-Omaha/Paradise/250e8ec7b4aa1f1f70c39c33b59819b1f2edeefd/sound/effects/powerloss.ogg when grid check starts.

Use firefox if the sound wont work for you

:cl:
add: Grid check event sound effect
/ :cl:

### CODE REVIEW

**Comment by Kyep** [comment]
Sound effect is neat.

**Comment by KasparoVy** [comment]
Hyper dramatic for when we get those really short poweroutages hahaha. Awesome all the same

File: code/modules/events/grid_check.dm [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 /datum/event/grid_check/start()
 	power_failure(0)
+	for(var/mob/M in GLOB.player_list)
+		if(!M.client)
+			continue
+		SEND_SOUND(M, sound('sound/effects/powerloss.ogg'))
```
Comment (Fox-McCloud) [inline code/modules/events/grid_check.dm]: Don't do this.

You're creating a sound for every single player in the world, multiple times.

cache it locally and send it to the necessary players. You also shouldn't be sending it to every single player in the world; it should be the same people who can hear the announcement.

Comment (KasparoVy) [reply code/modules/events/grid_check.dm]: Maybe a `playsound` `playsound_local` with a fucking immeeeennsseee range that ignores walls and emanates from the engine, wherever it is located on the map

Comment (Ty-Omaha) [reply code/modules/events/grid_check.dm]: okay so what I understand is make a var that has the sound and then send the var?

**Comment by Ty-Omaha** [comment]
updated @Fox-McCloud

File: code/modules/events/grid_check.dm [hunk 2]
```diff
@@ -1,16 +1,18 @@
 /datum/event/grid_check	//NOTE: Times are measured in master controller ticks!
 	announceWhen		= 5
-
+	var/power_loss_sound = sound('sound/effects/powerloss.ogg')
```
Comment (Fox-McCloud) [inline code/modules/events/grid_check.dm]: this should be `var/sound/s = sound('sound/effects/powerloss.ogg')`

and the whole thing should be done when `start` is called. We don't want to needlessly create sounds when we don't need them.

**Comment by Mitchs98** [comment]
> Use firefox if the sound wont work for you

:sweat:

**Comment by Spacemanspark** [comment]
Firefox is for nerds

**Comment by Ty-Omaha** [comment]
yeah I dont use it either lol

**Pull request merged**